monai/metrics/confusion_matrix.py

```diff
@@ -84,13 +84,15 @@ def _compute_tensor(self, y_pred: torch.Tensor, y: torch.Tensor):  # type: ignor
             ValueError: when `y` is not a binarized tensor.
             ValueError: when `y_pred` has less than two dimensions.
         """
-        if not isinstance(y_pred, torch.Tensor) or not isinstance(y, torch.Tensor):
-            raise ValueError("y_pred and y must be PyTorch Tensor.")
-        # check binarized input
-        if not torch.all(y_pred.byte() == y_pred):
-            warnings.warn("y_pred should be a binarized tensor.")
-        if not torch.all(y.byte() == y):
-            raise ValueError("y should be a binarized tensor.")
+        try:
+            is_binary_tensor(y_pred, "y_pred")
+        except ValueError as e:
+            raise ValueError("y_pred" + str(e)) from e
```

Comment:
cool, I think we can drop the try...except here:
```suggestion
        is_binary_tensor(y_pred, "y_pred")
```

Reply:
Yeah we can, my bad 😅